saleor/tests/e2e/account/test_should_create_account_without_email_confirmation.py

```diff
@@ -10,22 +9,29 @@
 def test_should_create_account_without_email_confirmation_core_1502(
     e2e_not_logged_api_client,
     e2e_staff_api_client,
+    permission_manage_products,
     permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+    permission_manage_taxes,
     permission_manage_settings,
 ):
     # Before
-    permissions = [permission_manage_channels, permission_manage_settings]
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_product_types_and_attributes,
+        permission_manage_shipping,
+        permission_manage_taxes,
+        permission_manage_settings,
+    ]
```

Comment:
If these are common and required whenever we use `prepare_shop`, maybe it would make sense to add a fixture `shop_permissions` that always returns this list, wdyt?